cypress-io/cypress
# fix: update net stubbing to not intercept requests sent to dev server

Fixes a potential hang when running components tests. See this [comment](https://github.com/cypress-io/cypress/issues/27769#issuecomment-1712380595) for more details.

### Additional details

* Updated the net-stubbing intercept logic to no longer match on requests meant for the dev server in CT.

### Steps to test

Set up a CT project with multiple specs:
* Spec 1 has a single test that intercepts all requests (`'*'`) and returns a static response.
* Spec 2 has another test

Run the component specs and verify Cypress does not hang.

### How has the user experience changed?

Cypress no longer hangs.

### PR Tasks


- [n/a] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [n/a] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
